content/release-notes/2022-09-28.md

```diff
@@ -9,13 +9,13 @@ label: 'Console'
 - Console: Added a `protected: boolean` flag to the roles public API response.
 - Console: Changed region and platform ids from serial numbers to strings in the public API.
 - Console: Changed the operations id from a serial number to a UUID type in the public API.
-- Control Plane: Added an internal mechanism for relocating projects between pageservers.
+- Control Plane: Added an internal mechanism for relocating projects between Pageservers.
 - Control Plane: Stopped sending metrics to Grafana Cloud.
 - Control Plane: Removed usage of the obsolete `zenith_ctl` binary.
 - Control Plane: Split the monolithic project object into project, branches, and endpoints in the internal database schema.
 
 ### Bug fixes
 
 - UI: Fixed a failure that occurred after receiving an empty response in the SQL Editor.
-- UI: Fixed an issue to ensure that table names are reset when switching between schemas in the Tables UI.
-- Control Plane: Fixed a stuck passwordless connection issue that occurred when creating a new project using a project selecting UI.
+- UI: Fixed an issue to ensure that table names are reset when switching between schemas on the **Tables** page in the Neon Console.
+- Control Plane: Fixed a passwordless connection issue that occurred when creating a new project using the project selection interface. Neon's passwordless auth feature allows you to quickly connect to a new or existing Neon project. For more information, see [Connect with Neon's psql passwordless auth](/docs/get-started-with-neon/query-with-psql-editor/#connect-with-neons-psql-passwordless-auth).
```

Comment:
```suggestion
- Control Plane: Fixed a [passwordless connection](/docs/get-started-with-neon/query-with-psql-editor/#connect-with-neons-psql-passwordless-auth) issue that occurred when creating a new project using the project selection interface.
```

Not sure if the Release notes should explain the features they refer to. Maybe a link is enough?